pkgs/development/libraries/libqglviewer/default.nix

```diff
@@ -1,28 +1,38 @@
-{ lib, stdenv, fetchurl, qmake, qtbase, libGLU, AGL }:
+{ lib, stdenv, fetchurl, qmake, qtbase, libGLU }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "libqglviewer";
   version = "2.9.1";
 
   src = fetchurl {
-    url = "http://www.libqglviewer.com/src/libQGLViewer-${version}.tar.gz";
+    url = "http://www.libqglviewer.com/src/libQGLViewer-${finalAttrs.version}.tar.gz";
     sha256 = "sha256-J4+DKgstPvvg1pUhGd+8YFh5C3oPGHaQmDfLZzzkP/M=";
   };
 
+  # Fix build on darwin, and install dylib instead of framework
+  prePatch = lib.optionalString stdenv.hostPlatform.isDarwin ''
+    substituteInPlace QGLViewer/QGLViewer.pro \
+      --replace-fail \
+        "LIB_DIR_ = /Library/Frameworks" \
+        "LIB_DIR_ = \$\$""{PREFIX_}/lib" \
+      --replace-fail \
+        "!staticlib: CONFIG *= lib_bundle" \
+        ""
+  '';
+
   nativeBuildInputs = [ qmake ];
-  buildInputs = [ qtbase libGLU ]
-    ++ lib.optional stdenv.hostPlatform.isDarwin AGL;
+  buildInputs = [ qtbase libGLU ];
 
   dontWrapQtApps = true;
 
   postPatch = ''
     cd QGLViewer
   '';
 
-  meta = with lib; {
+  meta = {
     description = "C++ library based on Qt that eases the creation of OpenGL 3D viewers";
     homepage = "http://libqglviewer.com";
```

Comment:
That points to russian spam site

Reply:
Ukrainian, which I mainly notice by the dotted "i"

Reply:
I think pointing to https://github.com/GillesDebunne/libQGLViewer as website would be acceptable for now, as long as we don't know about any actual vulnerabilities in the library itself